core/src/components/cat-select/cat-select.tsx

```diff
@@ -119,6 +119,7 @@ export class CatSelect {
   private subscription?: Subscription;
   private term$: Subject<string> = new Subject();
   private more$: Subject<void> = new Subject();
+  private isUserChanged = false;
```

Comment:
@geralrodrigues Why is this variable called isUserChanged? it could be any value, right?

Reply:
Only boolean, it's a flag that I am using to know if it is a change by the UI or programmatically. Basically, if it is programmatic, it goes through value first and then selection, and through the ui, first selection and then value

Reply:
Should a better name be used? Now I put the type too

Reply:
Well, I cannot put the type😅 
![image](https://user-images.githubusercontent.com/100929087/205278576-0431f7a9-d726-4cf7-a4e3-e65cfdf7c0a3.png)